libc-dev required on debian to build maddy

# Describe the bug

I noticed that building maddy with the build script fails on debian 10 if the g++ package isn't installed

# Steps to reproduce

1. Fresh VM with debian 10
2. `apt-get install gcc git curl`
3. `curl 'https://foxcpp.dev/maddy/build.sh' | bash`
This produces following error:
````
[...]
go: finding github.com/go-sql-driver/mysql v1.5.0
go: finding github.com/pierrec/lz4 v2.4.1+incompatible
# runtime/cgo
_cgo_export.c:3:10: fatal error: stdlib.h: No such file or directory
 #include <stdlib.h>
          ^~~~~~~~~~
compilation terminated.
````

Installing the `g++` package fixes this.

# Log files
[Full log](https://paste.sr.ht/%7Ebn4t/9967c914e41015bb0920e3a8ab7688c9fbcad6d5)

# Environment information
uname -a: `Linux mail 4.19.0-8-amd64 #1 SMP Debian 4.19.98-1 (2020-01-26) x86_64 GNU/Linux`

--- a/docs/tutorials/setting-up.md
+++ b/docs/tutorials/setting-up.md
@@ -29,10 +29,10 @@ there are a few things to keep in mind:
 Since there are currently no pre-compiled binaries for maddy, we are going to
 build it from the source. Nothing scary, this is relatively easy to do with Go.
 
-System dependencies you need to have installed is C compiler, Git and curl.
+System dependencies you need to have installed is C toolchain, Git and curl.
 On Debian-based distributions, this should be enough:
 ```
-# apt-get install gcc git curl
+# apt-get install gcc git curl libc6-dev
 ```
 
 If you want manual pages with reference docs, install scdoc too:
@@ -179,10 +179,10 @@ The email world is full of annoying bots just like Web (these annoying scanners
 looking for PhpMyAdmin on your blog). fail2ban can help you get rid of them by
 temporary blocking offending IPs.
 
-1. Install the fail2ban package using your distribution package manager
+1. Install the fail2ban package and the python systemd module using your distribution package manager
 For Debian-based distributions:
 ```
-apt-get install fail2ban
+apt-get install fail2ban python3-systemd
 ```
 
 2. build.sh already installed necessary jail configuration files, but you have to

